modules/tinymce/src/core/demo/ts/demo/TinyMceDemo.ts

```diff
@@ -19,10 +19,10 @@ export default function () {
     // images_upload_credentials: true,
     skin_url: '../../../../js/tinymce/skins/ui/oxide',
     setup(ed) {
-      ed.addButton('demoButton', {
+      ed.ui.registry.addButton('demoButton', {
         type: 'button',
```

Comment:
You could remove this if you want, as it does nothing in v5.

Reply:
Thanks! Done.